packages/dcos-integration-test/extra/test_networking.py

```diff
@@ -413,8 +397,16 @@ def vip_workload_test(dcos_api_session, container, vip_net, proxy_net, ipv6, nam
     else:
         origin_app = MarathonApp(container, vip_net, origin_host, vip, ipv6=ipv6, app_name_fmt=origin_fmt)
         proxy_app = MarathonApp(container, proxy_net, proxy_host, ipv6=ipv6, app_name_fmt=proxy_fmt)
+    # Port mappiong application runs on `proxy_host` and has a port mapping from a random port to `vip_port`.
```

Comment:
port mapping is for the backend servers. It has no relationship with vip_port. Maybe we could  remove `and has a port mapping from a random port to vip_port`

Reply:
Maybe the comment is confusing, but It has strong relationship with `vip_port`. `vip_port` is a port for vip label. Port mapping application sets `host_port` to `vip_port`.

Reply:
ah! I understand the comment now. Maybe we can say that `and has the host_port same as vip_port`